main.py

```diff
@@ -314,7 +315,7 @@ def updateServer(override):
 def runCommand(command):
     c.RCON.read()
     if c.RCON.enabled and c.RCON.password != '':
-        with Client('mieserver.ddns.net', c.RCON.port, passwd=c.RCON.password) as client:
+        with Client('minecraun.ddns.net', c.RCON.port, passwd=c.RCON.password) as client:
```

Comment:
I know that this was for testing, and that we are going to resolve this issue with one of the next cards, but until that time I think this should be for mieserver.ddns.net, don't you? Especially if we plan on updating it here soon.

Reply:
Given that that functionality is in this version, that seems really nitpicky... Regardless, it's been updated.